jenkins-pipelines/longevity-ics-nemesis-alter-table-1tb-7d.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    params: params,
+
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_ics_test.IcsLongevetyTest.test_ics_1_column_longevity',
+    test_config: 'test-cases/longevity/longevity-ics-nemesis-alter-table-1TB-7days.yaml',
+
+    timeout: [time: 10960, unit: 'MINUTES'],
+    post_behaviour: 'keep'
```

Comment:
remove this default `post_behaviour`, and use one of the new parameters if needed

Reply:
Done